spyder/plugins/editor/plugin.py

```diff
@@ -997,7 +996,6 @@ def get_plugin_actions(self):
                        self.mac_eol_action)
         add_actions(eol_action_group, eol_actions)
         eol_menu = SpyderMenu(parent=self, title=_("Convert end-of-line characters"))
-        eol_menu.setObjectName('checkbox-padding')
```

Comment:
This is not necessary anymore because PyQt 5.15 can render menus with checkboxes as expected.